packages/mobile/src/screens/sign-on-screen/screens/CreatePasswordScreen.tsx

```diff
@@ -58,34 +61,48 @@ export const CreatePasswordScreen = () => {
     >
       {({ handleSubmit: triggerSubmit, dirty, isValid }) => (
         <Page>
-          <Heading
-            heading={messages.createYourPassword}
-            description={messages.description}
-          />
-          <Flex direction='column' h='100%' gap='l'>
-            <ReadOnlyField label={messages.yourEmail} value={email} />
-            <TextField
-              name='password'
-              label={messages.passwordLabel}
-              textContentType='password'
-              secureTextEntry
-              noGutter
+          <KeyboardAvoidingView
+            keyboardShowingOffset={keyboardOffset - 16}
+            style={{ overflow: 'hidden' }}
+          >
+            <Heading
+              heading={messages.createYourPassword}
+              description={messages.description}
             />
-            <TextField
-              name='confirmPassword'
-              label={messages.confirmPasswordLabel}
-              textContentType='password'
-              secureTextEntry
-              noGutter
+            <Flex
+              direction='column'
+              h='100%'
+              gap='l'
+              onLayout={(e) => {
+                e.currentTarget.measureInWindow((x, y, w, h) => {
+                  setKeyboardOffset(y)
+                })
+              }}
+            >
+              <ReadOnlyField label={messages.yourEmail} value={email} />
+              <TextField
+                name='password'
+                label={messages.passwordLabel}
+                textContentType='password'
+                secureTextEntry
+                noGutter
+              />
+              <TextField
+                name='confirmPassword'
+                label={messages.confirmPasswordLabel}
+                textContentType='password'
+                secureTextEntry
+                noGutter
+              />
+              <PasswordCompletionChecklist />
+            </Flex>
+            <PageFooter
```

Comment:
in the video it appeared there was too much padding on the footer. was that like that before adding the keyboard avoiding view?

Reply:
oh actually there are two padding issues. i'll fix 👍